README.md

```diff
@@ -167,20 +140,32 @@ for i in range(1, 10):
     )
 ```
 
-Update an existing item:
+### Query the database
+
+A Cosmos DB SQL API database account supports querying the items in a container using SQL-like syntax.
+
+This example queries a container for items with a specific `id`:
 
 ```Python
-item = items[0]
-item['firstName'] = 'Some Other Name'
-updated_item = container.upsert_item(item)
+database = client.get_database(database_name)
+container = database.get_container(container_name)
+
+# Enumerate the returned items
+import json
+for item in container.query_items(query='SELECT * FROM mycontainer r WHERE r.id="something"'):
+    print(json.dumps(item, indent=True))
 ```
 
+> NOTE: Although you can specify *any* value for the container name in the `FROM` clause, we recommend you use the container name for consistency.
```

Comment:
One thing that bit me was when I used a reserved name for the alias (order in my case). This made the SQL query syntax invalid.

Reply:
Ah yes, good point! Will re-word on next pass.